bundles/org.openhab.binding.mercedesme/pom.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.openhab.addons.bundles</groupId>
+    <artifactId>org.openhab.addons.reactor.bundles</artifactId>
+    <version>3.4.0-SNAPSHOT</version>
+  </parent>
+
+  <dependencies>
+    <!-- https://mvnrepository.com/artifact/org.json/json -->
+    <dependency>
+      <groupId>org.json</groupId>
+      <artifactId>json</artifactId>
+      <version>20220320</version>
+    </dependency>
```

Comment:
Sure, I used GSON e.g. in my binding contribution MyBMW. It seems to be "well designed" and no problem parsing it with GSON. But in this case it causes me a headache, Please give me some rope here because I think the JSON format isn't well designed.

See Mercedes [response file status](https://github.com/openhab/openhab-addons/pull/13044/files#diff-d3120d8d126522afa5eb6f43ff8442c85469ea60f4e558038f10b5ec7ce4cd24)
It's an array of JSON Objects but every object has a different key. So in fact it isn't an array of the same type. Mapping this to GSON looks like a nightmare. Sure I can create a DTO class with all keys involved but afterwards I've to check again which key ist set. 

I checked all other bindings and org.json is included in several other projects.
So for me it looks more convinient to provide a very small Parser with Mapping table to get things done

Reply:
You should use the same version for the json library as the other bindings, otherwise you'll end up with class cast exceptions like this: https://community.openhab.org/t/openhab-cloud-class-org-json-jsonobject-cannot-be-cast-to-class-org-json-jsonobject/137971